src/convertToObject.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 
+const toCamelCase = require('./toCamelCase');
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const splitString = sourceString.split(';');
+  const newObject = {};
+
+  splitString.forEach((rule) => {
```

Comment:
The `forEach` loop can be replaced with a `reduce` function to build the new object. This will adhere to the checklist point about avoiding loops and using iteration methods instead.